iati-activities-schema.xsd

```diff
@@ -1122,20 +1110,7 @@
               </xsd:element>
               <xsd:any namespace="##other" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>
             </xsd:sequence>
-            <xsd:attribute name="ref" use="optional" type="xsd:string">
-              <xsd:annotation>
-                <xsd:documentation xml:lang="en">
-                  Machine-readable identification string for the
-                  organisation. Must be in the format {Registration
-                  Agency} - (Registration Number} where {Registration
-                  Agency} is a valid code in the Registration Agency
-                  code list and {Registration Number } is a valid
-                  identifier issued by the {Registration Agency}. If
-                  this is not present then the narrative MUST contain
-                  the name of the organisation.
-                </xsd:documentation>
-              </xsd:annotation>
-            </xsd:attribute>
+            <xsd:attribute ref="ref" use="optional"/>
```

Comment:
This replaces an attribute which does not an identical definition to the others. Suggest the change for this line is reverted...

Reply:
This is because this attribute definition has been changed to be the same as the others as detailed by @PetyaKangalova in the corresponding issue https://github.com/IATI/IATI-Schemas/issues/339

Reply:
Correct! Will dismiss that review and approve!